### PR TITLE
Allow setting throwable for CommandResult.

### PR DESCRIPTION
**Pull request by minecrafter**
[API](https://github.com/SpongePowered/SpongeAPI/pull/1075) | **Common**

Resolves SpongePowered/SpongeAPI#1073

### CODE REVIEW

**Comment by minecrafter** [comment]
I'm new to contributing, and this has an API change. Any guidance would be appreciated.

File: src/main/java/org/spongepowered/common/command/SpongeCommandManager.java [hunk 1]
```diff
@@ -289,6 +289,7 @@ public CommandResult process(CommandSource source, String commandLine) {
             source.sendMessage(error(t("Error occurred while executing command: %s", excBuilder.build())));
             this.log.error(TextSerializers.PLAIN.serialize(t("Error occurred while executing command '%s' for source %s: %s", commandLine, source.toString(), String
                     .valueOf(thr.getMessage()))), thr);
+            return CommandResult.build().throwable(thr).build();
```
Comment (dualspiral) [inline src/main/java/org/spongepowered/common/command/SpongeCommandManager.java]: Shouldn't this be `return CommandResult.builder().throwable(thr).build();`?

Comment (kashike) [inline src/main/java/org/spongepowered/common/command/SpongeCommandManager.java]: More than likely, yes.

**Comment by zml2008** [comment]
Closing, see API PR for explanation

**Pull request closed without merging**